src/gui/plugins/scene3d/Scene3D.cc

```diff
@@ -1970,8 +1970,9 @@ TextureNode::TextureNode(QQuickWindow *_window)
 #if QT_VERSION < QT_VERSION_CHECK(5, 14, 0)
   this->texture = this->window->createTextureFromId(0, QSize(1, 1));
 #else
+  int blank_id = 0;
```

Comment:
Nitpick: style is `camelCase`.

Also, we may want to match the `ign-gui` implementation from https://github.com/ignitionrobotics/ign-gui/pull/155/, so:

```suggestion
  void * nativeLayout;
```

Reply:
ah mb for the formatting,

yeah I figured it would be a good idea to keep the same implementation.